libs/remix-tests/src/run.ts

```diff
@@ -131,8 +131,12 @@ commander
     await provider.init()
     web3.setProvider(provider)
     extend(web3)
-    runTestFiles(path.resolve(file_path), isDirectory, web3, compilerConfig, (error) => {
-      if (error) process.exit(1)
+    runTestFiles(path.resolve(file_path), isDirectory, web3, compilerConfig, (error, totalPassing, totalFailing) => {
+      if (error) return process.exit(1)
+      if (totalFailing > 0) {
+        log.error(`\n\t${totalFailing} test${totalFailing > 1 ? 's' : ''} failed`)
+        process.exit(1)
```

Comment:
I think CI fails because unit tests interprets this as a unit test failure.. which isn't the case.